pkg/services/featuremgmt/registry.go

```diff
@@ -0,0 +1,162 @@
+package featuremgmt
+
+import "github.com/grafana/grafana/pkg/services/secrets"
+
+var (
+	FLAG_database_metrics = "database_metrics"
+	FLAG_live_config      = "live-config"
+
+	// Register each toggle here
+	standardFeatureFlags = []FeatureFlag{
+		{
+			Name:            "recordedQueries",
+			Description:     "Supports saving queries that can be scraped by prometheus",
+			State:           FeatureStateBeta,
+			RequiresLicense: true,
+		},
+		{
+			Name:            "teamsync",
+			Description:     "Team sync lets you set up synchronization between your auth providers teams and teams in Grafana",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/team-sync/",
+			RequiresLicense: true,
+		},
+		{
+			Name:            "ldapsync",
+			Description:     "Enhanced LDAP integration",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/enhanced_ldap/",
+			RequiresLicense: true,
+		},
+		{
+			Name:            "caching",
+			Description:     "Temporarily store data source query results.",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/query-caching/",
+			RequiresLicense: true,
+		},
+		{
+			Name:            "dspermissions",
+			Description:     "Data source permissions",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/docs/grafana/latest/enterprise/datasource_permissions/",
+			RequiresLicense: true,
+		},
+		{
+			Name:            "analytics",
+			Description:     "Analytics",
+			State:           FeatureStateStable,
+			RequiresLicense: true,
+		},
+		{
+			Name:            "enterprise.plugins",
+			Description:     "Enterprise plugins",
+			State:           FeatureStateStable,
+			DocsURL:         "https://grafana.com/grafana/plugins/?enterprise=1",
+			RequiresLicense: true,
+		},
+		{
+			Name:        "trimDefaults",
+			Description: "Use cue schema to remove values that will be applied automatically",
+			State:       FeatureStateBeta,
+		},
+		{
+			Name:        secrets.EnvelopeEncryptionFeatureToggle,
+			Description: "encrypt secrets",
+			State:       FeatureStateBeta,
+		},
+
+		{
+			Name:  "httpclientprovider_azure_auth",
+			State: FeatureStateBeta,
+		},
+		{
+			Name:            "service-accounts",
+			Description:     "support service accounts",
+			State:           FeatureStateBeta,
+			RequiresLicense: true,
+		},
+
+		{
+			Name:        FLAG_database_metrics,
+			Description: "Add prometheus metrics for database tables",
+			State:       FeatureStateStable,
+		},
+		{
+			Name:        "dashboardPreviews",
```

Comment:
Can we somehow force the style for all next feature toggles, for example use camelCase only 🤔 Maybe at least write the suggested name format in FeatureFlag.Name field?

Reply:
In my original draft, I tried to fix that all up... but decided to punt until future PRs :)